app/brave_settings_strings.grdp

```diff
@@ -281,9 +281,6 @@
     These are the default Shields settings. They apply to all websites unless you change something in the Shields panel on a particular site. Changing these won't affect your existing per-site settings.
   </message>
   <message name="IDS_SETTINGS_BRAVE_SHIELDS_DEFAULTS_DESCRIPTION_2" desc="The description for Brave shields Defaults sub-section in settings.">
-    This will block most ads on websites, not Brave Private Ads — those are controlled in the <ph name="BEGIN_LINK">&lt;a target="_blank" href="$1"&gt;</ph>Brave Rewards<ph name="END_LINK">&lt;/a&gt;</ph> settings.
-  </message>
-  <message name="IDS_SETTINGS_BRAVE_SHIELDS_DEFAULTS_DESCRIPTION_2_REWARDS_DISABLED" desc="The description for Brave shields Defaults sub-section in settings when Brave Rewards is disabled.">
     This will block most ads on websites.
```

Comment:
@petemill @mkarolin basically i've renamed IDS_SETTINGS_BRAVE_SHIELDS_DEFAULTS_DESCRIPTION_2_REWARDS_DISABLED to IDS_SETTINGS_BRAVE_SHIELDS_DEFAULTS_DESCRIPTION_2, because the text will now be the same regardless of whether rewards is enabled. is there a way to remap the translations accordingly since this doesn't actually require new translations?

Reply:
The translations should be unaffected. I think they're based on string value and not message name.

Reply:
Even if it causes a change in transifex the translations should be picked up from the translation memory.